website/docs/d/cd_tekton_pipeline.html.markdown

```diff
@@ -2,185 +2,184 @@
 layout: "ibm"
 page_title: "IBM : ibm_cd_tekton_pipeline"
 description: |-
-  Get information about tekton_pipeline
+  Get information about cd_tekton_pipeline
 subcategory: "CD Tekton Pipeline"
 ---
 
 # ibm_cd_tekton_pipeline
 
-~> **Beta:** This data source is in Beta, and is subject to change.
-
-Provides a read-only data source for tekton_pipeline. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
+Provides a read-only data source for cd_tekton_pipeline. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
 
 ## Example Usage
 
 ```hcl
-data "ibm_cd_tekton_pipeline" "tekton_pipeline" {
-	pipeline_id = "94619026-912b-4d92-8f51-6c74f0692d90"
+data "ibm_cd_tekton_pipeline" "cd_tekton_pipeline" {
+	id = "94619026-912b-4d92-8f51-6c74f0692d90"
 }
 ```
 
 ## Argument Reference
 
 Review the argument reference that you can specify for your data source.
 
-* `pipeline_id` - (Required, Forces new resource, String) ID of current instance.
+* `id` - (Required, Forces new resource, String) ID of current instance.
   * Constraints: The maximum length is `36` characters. The minimum length is `36` characters. The value must match regular expression `/^[-0-9a-z]+$/`.
 
 ## Attribute Reference
 
 In addition to all argument references listed, you can access the following attribute references after your data source is created.
 
-* `build_number` - (Optional, Integer) The latest pipeline run build number. If this property is absent, the pipeline hasn't had any pipelineRuns.
```

Comment:
Again all attibute reference can't have Required/optional ..They are just computed in all docs

Reply:
Not sure I follow, can you clarify your meaning?

Reply:
build_number no longer has Optional param in the doc